.github/workflows/staging-cd.yml

```
@@ -0,0 +1,92 @@
+          credentials_json: '${{ secrets.GCP_STAG_SA_KEY }}'
+
+      # GCP SDK 설치 및 인증 정보 설정
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@v2
+
+      # GCP Artifact Registry 인증
+      - name: Authenticate Docker with GCP Artifact Registry
+        run: gcloud auth configure-docker ${{ env.REGION }}
+
+      - name: Set IMAGE variable
+        run: echo "IMAGE=${{ env.REGION }}/${{ secrets.GCP_STAG_PROJECT_ID }}/${{ env.IMAGE_REGISTRY }}/fitrun-staging" >> $GITHUB_ENV
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:111: Double quote to prevent globbing and word splitting

(shellcheck)